demoapp/composeApp/src/commonMain/kotlin/at/asitplus/cryptotest/App.kt

```diff
@@ -112,22 +112,29 @@ internal fun App() {
             X509SignatureAlgorithm.RS1,
             X509SignatureAlgorithm.RS256,
             X509SignatureAlgorithm.RS384,
-            X509SignatureAlgorithm.RS512)
-        var keyAlgorithm by remember { mutableStateOf<SpecializedSignatureAlgorithm>(X509SignatureAlgorithm.ES256) }
+            X509SignatureAlgorithm.RS512
+        )
+        var keyAlgorithm by remember {
+            mutableStateOf<SpecializedSignatureAlgorithm>(
+                X509SignatureAlgorithm.ES256
+            )
+        }
```

Comment:
this seems like it's just adding a bunch of extra whitespace

Reply:
line-length auto-formatter